cli/args.go

```diff
@@ -257,6 +264,21 @@ func parseStringArg(args []string, argName string, defaultValue string) (string,
 	return defaultValue, nil
 }
 
+// Find a int argument (e.g. --foo 1) of the given name in the given list of arguments. If it's present,
+// return its value. If it is present, but has no value, return an error. If it isn't present, return defaultValue.
+func parseIntArg(args []string, argName string, defaultValue string) (int, error) {
```

Comment:
Shouldn't the `defaultValue` already be an `int`?

Reply:
`defaultValue` gets read in by `os.GetEnv` [here](https://github.com/gruntwork-io/terragrunt/pull/636/files/88fb562714555232b161799148c018386abcc351#diff-cfc2a8f0c663e6f451ddbff25d8cffd9R106), so i thought it would be ok to pass in the string to keep consistency and do the `Atoi` if the default value was used to return